### PR TITLE
Change seed to viewing key - Typo fixed.

### PR DESCRIPTION
**Pull request by juanky201271**

### CODE REVIEW

File: app/translations/en.json [hunk 1]
```diff
@@ -432,7 +432,10 @@
     "tapcopy-message": "Copied Key to Clipboard",
     "title": "Key",
     "address": "Address",
-    "text-readonly": "This is your viewing key. Please write/copy it down carefully. It is the only way to restore your actual wallet."
+    "text-readonly": "This is your viewing key. Please write/copy it down carefully. It is the only way to restore your actual wallet.",
```
Comment (zancas) [inline app/translations/en.json]: The viewing key doesn't restore spend authority.   So maybe we should change the second sentence to:

`Anyone who has a copy of the viewing key can observe all transactions to and from the associated wallet.`

File: app/translations/en.json [hunk 2]
```diff
@@ -432,7 +432,10 @@
...
+    "change-warning": "IF YOU CHANGE TO ANOTHER WALLET, YOU WILL NO LONGER HAVE ACCESS TO THIS WALLET WITHOUT THE VIEWING KEY.",
```
Comment (zancas) [inline app/translations/en.json]: "HAVE ACCESS TO" ---> "BE ABLE TO VIEW THIS WALLET"

File: app/translations/en.json [hunk 3]
```diff
@@ -432,7 +432,10 @@
...
+    "backup-warning": "IF YOU CHANGE TO YOUR BACKUP WALLET, YOU WILL NO LONGER HAVE ACCESS TO THIS WALLET WITHOUT THE VIEWING KEY.",
```
Comment (zancas) [inline app/translations/en.json]: "HAVE ACCESS TO" --> "BE ABLE TO VIEW"

File: app/translations/en.json [hunk 4]
```diff
@@ -432,7 +432,10 @@
...
+    "server-warning": "IF YOU CHANGE TO A SERVER WITH A DIFFERENT CHAIN, YOU WILL NO LONGER HAVE ACCESS TO THIS WALLET WITHOUT THE VIEWING KEY"
```
Comment (zancas) [inline app/translations/en.json]: There's a 1:1 map from wallet to chain..     so we should say:

"THE SERVER IS TRUSTED TO REPORT THE PUBLIC BLOCKCHAIN.  ONLY SELECT A SERVER YOU TRUST!"

File: components/Ufvk/ShowUfvk.tsx [hunk 5]
```diff
@@ -80,17 +80,17 @@ const ShowUfvk: React.FunctionComponent<ShowUfvkProps> = ({ onClickOK, onClickCa
 
         {times === 3 && action === 'change' && (
           <FadeText style={{ padding: 20, textAlign: 'center', color: 'white' }}>
-            {translate('seed.change-warning') as string}
+            {translate('privkey.change-warning') as string}
```
Comment (zancas) [inline components/Ufvk/ShowUfvk.tsx]: The ident `privkey` is confusing.   Which key is the `privkey`?

File: app/translations/en.json (hunk 3 above)
Comment (juanky201271) [reply app/translations/en.json]: I think this is a duplicate comment...

File: app/translations/en.json (hunk 4 above)
Comment (juanky201271) [reply app/translations/en.json]: The context for this message is: The user changed the server, and with the new server the App is not be able to open the current wallet... (can happen for different things, but the most likely is that the user specify different `chain` of the server)... I don't know how to mix your suggestion and the existing message...

**Review by fluidvanadium — APPROVED** [review]
looks good

File: app/translations/en.json (hunk 4 above)
Comment (fluidvanadium) [reply app/translations/en.json]: juanky is right.

**Pull request merged**